Annotation/polar_golden_set.json

```diff
@@ -1297,13 +1430,13 @@
             {
                 "predicate": [
                     {
-                        "str": "tell about",
+                        "str": "tell",

```

Comment:
I guess it's ambiguous but I'd interpret it as "tell about this and tell about whether..."

Reply:
yeh it is ambiguous, I think I favoured deleting purely because the recipient is splitting the verb from the prep but I'm happy with either? I do worry about our tool and when the prep is separated from the verb?

Reply:
Both are fine - good point about the tool, it might be difficult to define heuristics for that. I'm not sure if we're at that stage of fine-tuning the code yet?